### PR TITLE
[Focus Mode] New images and content for the Performance reference article

### PR DESCRIPTION
**Pull request by captainbrosset**
New Focus Mode images and content for the **Performance features reference** article.

I shortened the image names as well, since they were extra long in this folder.

* Before: https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/evaluate-performance/reference
* After: https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/evaluate-performance/reference?branch=pr-en-us-2945

Also, article section:
* Before: https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/rendering-tools/#paint-and-composite-tools
* After: https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/rendering-tools/?branch=pr-en-us-2945#paint-and-composite-tools

AB#47861566

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 1]
```diff
@@ -374,56 +398,46 @@ Click an interaction to view more information about it in the **Summary** tab.
...
 
-DevTools provides numerous ways to analyze frames per second:
+DevTools provides two ways to analyze frames per second:
 
-*  Use [the FPS chart](#the-fps-chart) to get an overview of FPS over the duration of the recording.
 *  Use [the Frames section](#the-frames-section) to view how long a particular frame took.
 *  Use the **FPS meter** for a realtime estimate of FPS as the page runs.  See [View frames per second in realtime with the FPS meter](#view-frames-per-second-in-realtime-with-the-fps-meter).
 
 
-###### The FPS chart
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:384]: This got removed from DevTools, see https://bugs.chromium.org/p/chromium/issues/detail?id=1505063

File: microsoft-edge/devtools-guide-chromium/rendering-tools/index.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ Compositing is where the painted parts of the page are put together for displayi
 <!-- ------------------------------ -->
 #### Paint and composite: Tools
 
-Want to know how long painting takes or how often painting occurs?  Check the [Enable advanced paint instrumentation](../evaluate-performance/reference.md#turn-on-advanced-paint-instrumentation) setting in the **Performance** panel and then take a recording.  If most of your rendering time is spent painting, you have paint problems.
+Want to know how long painting takes or how often painting occurs?  Check the [Enable advanced rendering instrumentation](../evaluate-performance/reference.md#turn-on-advanced-rendering-instrumentation) setting in the **Performance** panel and then take a recording.  If most of your rendering time is spent painting, you have paint problems.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/rendering-tools/index.md:175]:
```suggestion
Want to know how long painting takes or how often painting occurs?  Select the [Enable advanced rendering instrumentation](../evaluate-performance/reference.md#turn-on-advanced-rendering-instrumentation) setting in the **Performance** panel, and then take a recording.  If most of your rendering time is spent painting, you have paint problems.
```

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 3]
```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
...
 
-To learn how to interact with the paint information, see [View layers](#view-layers-information) and [View paint profiler](#view-paint-profiler).
+See [View layers](#view-layers-information), and [View CSS selector statistics](#view-css-selector-statistics), below.
 
 
 <!-- ====================================================================== -->
 ## Save a recording
 
-To save a recording, right-click the recording and then select **Save Profile**.
+To save a recording as a file on your device, right-click the recording and then select **Save Profile**:
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:173]:
```suggestion
To save a recording as a file on your device, right-click the recording and then select **Save profile**:
```
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/reference?branch=pr-en-us-2945#save-a-recording

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 4]
```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
...
 
-![Save Profile](./reference-images/evaluate-performance-performance-refreshed-disable-javascript-samples-checkbox-off-save-profile.png)
+![Save Profile](./reference-images/save-profile.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:175]:
```suggestion
![The 'Save profile' right-click menuitem](./reference-images/save-profile.png)
```

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 5]
```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
...
 
 
 <!-- ====================================================================== -->
 ## Load a recording
 
-To load a recording, right-click the recording and then select **Load Profile**.
+To load a recording from a file, right-click in the **Performance** tool and then select **Load Profile**:
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:181]:
```suggestion
To load a recording from a file, right-click in the **Performance** tool and then select **Load profile**:
```

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 6]
```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
...
+
+![Load Profile](./reference-images/load-profile.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:183]:
```suggestion
![The 'Load profile' right-click menuitem](./reference-images/load-profile.png)
```

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 7]
```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
...
 
-1. Hover your mouse over the **Overview** section or the **Details** section.  The **Overview** section is the area containing the **FPS**, **CPU**, and **NET** charts.  The **Details** section is the area containing the **Main** section, the **Interactions** section, and so on.
 
-1. Using two fingers, swipe up to zoom out, swipe left to move left, swipe down to zoom in, and swipe right to move right.
+###### Trackpad
 
-To scroll a long flame chart in the **Main** section or any of the neighbors, click and hold while dragging up and down.  Drag left and right to move what portion of the recording is selected.
+To select a portion by using a trackpad:
+
+1. Hover your mouse over the **Overview** section or the **Details** section.  The **Overview** section is the area containing the **FPS**, **CPU**, and **NET** charts.  The **Details** section is the area containing the **Main** section and the **Interactions** section.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:226]: move paragraph to the Mouse section that i created above?

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 8]
```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
...
+
+1. Using two fingers, swipe up to zoom out, or down to zoom in.  Using two fingers, swipe left to move left, or right to move right.
+
+To scroll a long flame chart in the **Main** section or any of the neighbors, click and hold while dragging up and down.  Drag left and right to move which portion of the recording is selected.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:230]: 2. move paragraph to the Mouse section that i created above?

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:230]: moved

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md (hunk 7 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:226]: no; instead, i struck "your mouse"

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:384]: "fps chart" no longer is in article

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md [hunk 9]
```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
...
-![More information about the anonymous function in the Summary tab](./reference-images/evaluate-performance-performance-summary-me.png)
+![More information about the anonymous function in the Summary tab](./reference-images/summary-me.png)
 
 DevTools represents main thread activity with a flame chart.  The x-axis represents the recording over time.  The y-axis represents the call stack.  The events on top cause the events below it.
 
-![A flame chart](./reference-images/evaluate-performance-performance-main-flame-chart.png)
+![A flame chart](./reference-images/main-flame-chart.png)
 
-In the previous figure, a `click` event caused a `Function Call` in `activitytabs.js` on line 53.  Below `Function Call`, review that an anonymous function was run.  The anonymous function requested `a`, which requested `wait`, which requested `Minor GC`.
+In the previous figure, an `input` event caused a `Function Call`, which ran an anonymous function that called `filterByCamera` and then `populateGallery`.  The `populateGallery` function then made DOM changes by calling `innerHTML`.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md:273]:
```suggestion
In the previous figure, an `input` event caused a `Function Call`, which ran an anonymous function that called `filterByCamera` and then `populateGallery`.  The `populateGallery` function then made DOM changes by calling `setInnerHTML`.
```
@captainbrosset fyi (i'm editing in VS Code)
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/reference?branch=pr-en-us-2945#view-main-thread-activity

**Pull request merged**